libraries/SuplaDevice/supla/network/esp_wifi.h

```diff
@@ -73,7 +98,27 @@ class ESPWifi : public Supla::Network {
   }
 
   bool connect(const char *server, int port) {
-    return client.connect(server, port);
+    
+	bool result = client.connect(server, port);
+	
+#ifdef ESP_SSL
+	/* server certificate validation */
+	if (result && fingerprint)	{
+	  if (client.verify(fingerprint, server)) {
+#ifdef SUPLA_COMM_DEBUG
```

Comment:
I use this define to enable logging of bytes send to and from server
Here you can remove this ifdef and just print this log always.